vitrina/smart_contracts/urls.py

```diff
@@ -22,4 +24,14 @@
         AgreementDetailView.as_view(),
         name="agreement-detail",
     ),
+    path(
+        "projects/<int:pk>/agreement/<uuid:agreement_id>/generate_pdf/",
+        AgreementGeneratePdf.as_view(),
+        name="agreement-generate-pdf",
+    ),
+    path(
+        "projects/<int:pk>/agreement/<uuid:agreement_id>/upload_signed/",
+        AgreementUploadSignedFile.as_view(),
+        name="agreement-upload-signed-adoc",
```

Comment:
```suggestion
        "projects/<int:pk>/agreement/<uuid:agreement_id>/generate-pdf/",
        AgreementGeneratePdf.as_view(),
        name="agreement-generate-pdf",
    ),
    path(
        "projects/<int:pk>/agreement/<uuid:agreement_id>/upload-signed/",
        AgreementUploadSignedFile.as_view(),
        name="agreement-upload-signed-adoc",
```

is considered better practice and better overall url design.